solution/backend/package-lock.json

```diff
@@ -2386,20 +2385,6 @@
         "node": ">=6"
       }
     },
-    "node_modules/fsevents": {
-      "version": "2.3.2",
-      "resolved": "https://registry.npmjs.org/fsevents/-/fsevents-2.3.2.tgz",
-      "integrity": "sha512-xiqMQR4xAeHTuB9uWm+fFRcIOgKBMiOBP+eXiyT7jsgVCq1bkVygt00oASowB7EdtpOHaaPgKt812P9ab+DDKA==",
```

Comment:
what are fsevents? Is this related to changesets?

Reply:
Thats an error thing for node nothing to do with this.